src/workerd/jsg/modules-new-test.c++

```diff
@@ -1836,32 +1836,40 @@ KJ_TEST("Aliased modules (import maps) work") {
 
 // ======================================================================================
 
+// TODO(soon): Temporarily disabling this test. TC-39 recently changed the syntax
+// for import attributes to move away from the "assert". Prior to v8 12.6 the
+// assert keyword was still supporteed, with 12.6 it is not. Since we currently
```

Comment:
FYI we have used V8_MAJOR_VERSION/V8_MINOR_VERSION before to support both a newer and an older V8 version before; this can go ahead as-is though with the test not being used for production code.

Reply:
yep, I intentionally wanted to keep this simple, especially since it's still considered an experimental language feature and the test is non-critical. The test will need to be updated anyway so I'll do it all as part of the 12.6 update specifically for workerd.